src/graphql/models/UserAwardedBadge.js

```diff
@@ -0,0 +1,12 @@
+import { GraphQLBoolean, GraphQLObjectType, GraphQLString } from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'UserAwardedBadge',
+  fields: () => ({
+    badgeId: { type: GraphQLString },
+    badgeMetaData: { type: GraphQLString },
+    isDisplayed: { type: GraphQLBoolean },
+    createdAt: { type: GraphQLString },
+    updatedAt: { type: GraphQLString },
```

Comment:
Given the fact that all these fields are required (not optional) in [schema](https://github.com/cofacts/rumors-db/blob/master/schema/users.ts#L51-L55), we can safely declare that these fields are non-null.

This help reduce unnecessary null handling in API clients like rumors-site.

```suggestion
    badgeId: { type: new GraphQLNonNull(GraphQLString) },
    badgeMetaData: { type: new GraphQLNonNull(GraphQLString) },
    isDisplayed: { type: new GraphQLNonNull(GraphQLBoolean) },
    createdAt: { type: new GraphQLNonNull(GraphQLString) },
    updatedAt: { type: new GraphQLNonNull(GraphQLString) },
```